pkg/apis/kops/networking.go

```diff
@@ -86,6 +86,9 @@ type WeaveNetworkingSpec struct {
 	NPCCPULimit *resource.Quantity `json:"npcCPULimit,omitempty"`
 	// NPCExtraArgs are extra arguments that are passed to weave-npc.
 	NPCExtraArgs string `json:"npcExtraArgs,omitempty"`
+
+	// Version overrides the default tag of Weave container image.
```

Comment:
I would suggest de-emphasizing the default. Something like "Version specifies the Weave container image tag. The default depends on the kOps version."

Reply:
thanks for your suggestion @johngmyers , pushed changes accordingly ^^